### PR TITLE
Inputs

### PR DESCRIPTION
**Pull request by JasonLantz**
this implements different states for the inputs.... states are disabled, read-only, default, and error...

### CODE REVIEW

File: build/demo.compounds.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...f window||!window.document||!window.document.createElement),t.addEventListener=function(e,t,n){return e.addEventListener?e.addEventListener(t,n,!1):e.attachEvent("on"+t,n)},t.removeEventListener=function(e,t,n){return e.removeEventListener?e.removeEventListener(t,n,!1):e.detachEvent("on"+t,n)},t.getConfirmation=function(e,t){return t(window.confirm(e))},t.supportsHistory=function(){var e=window.navigator.userAgent;return(e.indexOf("Android 2.")===-1&&e.indexOf("Android 4.0")===-1||e.indexOf("Mobile Safari")===-1||e.indexOf("Chrome")!==-1||e.indexOf("Windows Phone")!==-1)&&(window.history&&"pushState"in window.history)},t.supportsPopStateOnHashChange=function(){return window.navigator.userAgent.indexOf("Trident")===-1},t.supportsGoWithoutReloadUsingHash=function(){return window.navigator.userAgent.indexOf("Firefox")===-1},t.isExtraneousPopstateEvent=function(e){return void 0===e.state&&navigator.userAgent.indexOf("CriOS")===-1}},"./node_modules/history/LocationUtils.js":function(e,t,n){"use strict";function a(e){return e&&e.__esModule?e:{default:e}}t.__esModule=!0,t.locationsAreEqual=t.createLocation=void 0;var r=Object.assign||function(e){for(var t=1;t<arguments.length;t++){var n=a
```
Comment (wyseguyonline) [inline build/demo.compounds.js]: Shouldn't these files be ignored?

Comment (JasonLantz) [inline build/demo.compounds.js]: we kept them in because i didn't have a good solution for the gh-pages...

i can comment them out.

**Comment by wyseguyonline** [comment]
I forgot we left it that way for the gh pages.  Go ahead and leave it.

File: README.md [hunk 2]
```diff
@@ -124,11 +133,11 @@ Note that re-installing node_modules will remove all symlinks.
 ## Documentation Site
 
 The static demo site is located at:
-http://localhost:8081/
+http://localhost:8081/compounds/
```
Comment (StommePoes) [inline README.md]: let's http--> https, or we'll certainly get PDA Help room complaints "it doesn't show anything in my browser locally."

Comment (JasonLantz) [inline README.md]: i changed it back to http... you can see the designated protocol in the webpack.config.js under dev server....

**Pull request merged**